pkg/provision/providers/docker/node.go

```diff
@@ -101,7 +101,8 @@ func (p *provisioner) createNode(ctx context.Context, clusterReq provision.Clust
 		})
 	}
 
```

Comment:
udev doesn't matter in docker, right?

Reply:
doesn't really, but Talos still does the write and fails